lib/sisimai/bite/email/apachejames.rb

```diff
@@ -0,0 +1,169 @@
+module Sisimai::Bite::Email
+  module ApacheJames
+    # Sisimai::Bite::::Email::ApacheJames parses a bounce email which created by
+    # ApacheJames. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/ApacheJames.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'subject'    => %r/\A\[BOUNCE\]\z/,
+        :'received'   => %r/JAMES SMTP Server/,
+        :'message-id' => %r/\d+[.]JavaMail[.].+[@]/,
+      }
+      Re1 = {
+        # apache-james-2.3.2/src/java/org/apache/james/transport/mailets/
+        #   AbstractNotify.java|124:  out.println("Error message below:");
+        #   AbstractNotify.java|128:  out.println("Message details:");
+        :begin  => %r/\AContent-Disposition:[ ]inline/,
+        :error  => %r/\AError message below:\z/,
+        :rfc822 => %r|\AContent-Type: message/rfc822|,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Java Apache Mail Enterprise Server'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
```

Comment:
__[rubocop]__
* Layout/EmptyLineBetweenDefs: Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->